emcc.py

```diff
@@ -1315,9 +1331,15 @@ def run(args):
     logger.debug('stopping after linking to object file')
     return 0
 
+  js_syms = {}
+  if not settings.SIDE_MODULE:
+    js_syms = get_all_js_syms()
+    deps_info.append_deps_info(js_syms)
+    logger.debug("JS symbols: " + pprint.pformat(js_syms))
```

Comment:
These calls to `pprint` happen unconditionally even if the output isn't logged. That might be wasteful, what do you think?

Reply:
Removed